randovania/gui/preset_settings/generation_tab.py

```diff
@@ -35,6 +36,11 @@ def __init__(self, editor: PresetEditor, game_description: GameDescription, wind
         signal_handling.on_checked(self.check_major_minor, self._persist_major_minor)
         signal_handling.on_checked(self.local_first_progression_check, self._persist_local_first_progression)
 
+        self.logical_pickup_placement_combo.setItemData(0, LogicalPickupPlacementConfiguration.MINIMAL)
```

Comment:
nothing for you to fix right now, but this feels like we should have a helper function for this at one point.

Reply:
for the moment, at least do:
```py
for i, logical_placement_mode in enumerate(LogicalPickupPlacementConfiguration):
    self.logical_pickup_placement_combo.setItemData(i, logical_placement_mode)
```